api/spec/src/productcatalog/routes.tsp

```diff
@@ -267,9 +251,10 @@ interface Subscriptions {
   @operationId("createSubscription")
   @summary("Create subscription")
   @extension("x-internal", true)
-  create(
-    @body body: SubscriptionCreate,
-  ): Subscription | OpenMeter.ConflictError | OpenMeter.CommonErrors;
+  create(@body body: SubscriptionCreate): {
+    @statusCode _: 201;
+    @body body: Subscription;
```

Comment:
Would it make sense to make `name` optional or omitted in `PlanSubscriptionChange` and use the plan's name by default? Also, currency should come from plan and could be removed.